plugins/rbac-backend/src/service/policy-builder.ts

```diff
@@ -71,11 +78,25 @@ export class PolicyBuilder {
       policy: await RBACPermissionPolicy.build(env.logger, env.config, enf),
     };
 
+    const pluginIdsConfig = env.config.getOptionalStringArray(
+      'permission.rbac.pluginsWithPermission',
+    );
+    if (pluginIdsConfig) {
+      const pluginIds = pluginIdProvider.getPluginIds();
+      pluginIdProvider.getPluginIds = () => {
+        return [...pluginIdsConfig, ...pluginIds];
+      };
```

Comment:
This would just combine the two. Would it be better to filter to ensure that a plugin isn't in both lists. I.e. a user defines the catalog plugin in `app-config`.